R/bfastts.R

```diff
@@ -197,7 +197,7 @@ bfastts <- function(data, dates, type = c("irregular", "16-day", "10-day")) {
   
   #  crop time series and return
   bounds = range(which(!is.na(A)))
-  return(ts(A[bounds[1]:bounds[2]], start=time(A)[bounds[1]], end=time(A)[bounds[2]], freq=freq))
+  return(ts(A[bounds[1]:bounds[2]], start=time(A)[bounds[1]], end=time(A)[bounds[2]], frequency=freq))
```

Comment:
@appelmar Is there any particular reason why you are doing this, instead of `window(A, start=bounds[1], end=bounds[2])`?

Reply:
no, see af125c9